src/components/feedback/Dialog.tsx

```diff
@@ -169,7 +169,7 @@ const DialogNext = function Dialog({
         buttons={buttons}
         fullWidthHeader={true}
         icon={icon}
-        onClose={closeHandler}
+        onClose={onClose}
```

Comment:
This isn't _exactly_ a substantive change but is purer in function for non-closeable dialogs (previously-existing legacy "feature").